packages/frontend-api/src/Model/Resolver/Transport/TransportResolver.php

```diff
@@ -41,7 +41,7 @@ public function __construct(TransportFacade $transportFacade, Domain $domain)
     public function resolver(string $uuid): Transport
     {
         try {
-            return $this->transportFacade->getByUuid($uuid);
+            return $this->transportFacade->getVisibleByUuid($uuid);
```

Comment:
Shouldn't be `TransportCanBeUsedValidator` updated too?

Reply:
Yes, nice catch!